pkg/issuer/venafi/client/venaficlient_test.go

```diff
@@ -223,6 +224,9 @@ type testConfigForIssuerT struct {
 }
 
 func (c *testConfigForIssuerT) runTest(t *testing.T) {
+	// Set issuer defaults
+	webhook.Scheme.Default(c.iss)
```

Comment:
This defaulting will mutate the argument (`c.iss`) - is that an issue? Is this structured re-used for later tests? (i.e. a shared base structure?)

Reply:
I've removed this.